project/src/main/java/com/google/autograder/servlets/manageBoxServlet.java

```diff
@@ -36,59 +37,50 @@
 @WebServlet("/manageBox")
 public class manageBoxServlet extends HttpServlet {
   
+  private Database database = new Database(); // should this be an existing database from elsewhere?
```

Comment:
Let's chat about this. We need to make sure we only use one Database/Datastore.

Right now I'm using Services.DATA_STORE for storage and maybe we can hook up Mahima's Database class to be a wrapper around that or to use that as it's datastore.